infra/src/test/kotlin/fi/fta/geoviite/infra/split/SplitServiceIT.kt

```diff
@@ -0,0 +1,65 @@
+package fi.fta.geoviite.infra.split
+
+import fi.fta.geoviite.infra.DBTestBase
+import fi.fta.geoviite.infra.common.IntId
+import fi.fta.geoviite.infra.math.Point
+import fi.fta.geoviite.infra.tracklayout.alignment
+import fi.fta.geoviite.infra.tracklayout.locationTrack
+import fi.fta.geoviite.infra.tracklayout.referenceLine
+import fi.fta.geoviite.infra.tracklayout.segment
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import kotlin.test.assertEquals
+
+@ActiveProfiles("dev", "test")
+@SpringBootTest
+class SplitServiceIT @Autowired constructor(
+    val splitService: SplitService,
+    val splitDao: SplitDao,
+) : DBTestBase() {
+
+
+    @Test
+    fun `should find splits by source location track`() {
```

Comment:
Tykkäisin itse että kun näissä nimissä kerran käytetään ihan kokonaisia sanoja niin se olisi myös kokonainen lause. Näistä mitkä alkaa "should x" jää aina mieleen että "what should?". Eli mielummin nimeäisin vaikkapa `findUnfinishedSplits filters by locationtrack` tjsp.